cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -425,21 +505,23 @@ func selectNodePool(workerType string, nodePools []astro.NodePool, out io.Writer
 // user gets prompted if no name for the queue to delete was specified
 // An errQueueDoesNotExist is returned if queue to delete does not exist
 // An errCannotDeleteDefaultQueue is returned if a user chooses the default queue
-func Delete(ws, deploymentID, deploymentName, name string, force bool, client astro.Client, coreClient astrocore.CoreClient, out io.Writer) error {
+func Delete(ws, deploymentID, deploymentName, name string, force bool, platformCoreClient astroplatformcore.CoreClient, coreClient astrocore.CoreClient, out io.Writer) error { //nolint:gocognit
 	var (
-		requestedDeployment          astro.Deployment
-		err                          error
-		queueToDelete                *astro.WorkerQueue
-		listToDelete, existingQueues []astro.WorkerQueue
-		queue                        astro.WorkerQueue
+		requestedDeployment astroplatformcore.Deployment
+		err                 error
+		queueToDelete       *astroplatformcore.WorkerQueueRequest
+		queueToDeleteHybrid *astroplatformcore.HybridWorkerQueueRequest
+		existingQueues      []astroplatformcore.WorkerQueue
+		listToDelete        []astroplatformcore.WorkerQueueRequest
+		hybridListToDelete  []astroplatformcore.HybridWorkerQueueRequest
```

Comment:
these should probably be called `workerQueuesToKeep` and `hybridWorkerQueuesToKeep`

Reply:
fixed